src/routes/Player/Player.js

```diff
@@ -237,6 +240,7 @@ const Player = ({ urlParams, queryParams }) => {
         if (!event.nativeEvent.subtitlesMenuClosePrevented) {
             closeSubtitlesMenu();
         }
+        closeAudioMenu();
```

Comment:
general now this does not work in some cases when you for example open the subtitles menu and click the audio menu button, both menu will be visible on top of each other